src/data/accessibility-quiz.ts

```diff
@@ -378,14 +378,14 @@ const accessibilityQuiz = [
   },
   {
     Question:
-      "Which of the following HTML tags does NOT use the autocomplete attribute?",
-    Answer: "<html>",
+      "Which of the following HTML tags does NOT use the rel attribute?",
+    Answer: "<link>",
     Distractor1: "<input>",
```

Comment:
it looks like input does not accept the rel attribute according to MDN

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input

```suggestion
    Distractor1: "<link>",
```

Reply:
@jdwilkin4 thank you fixed it